docs/appendix/contributing.md

```diff
@@ -52,6 +52,131 @@ When in doubt, ask during review.
 All documentation is in the [docs folder](https://github.com/viamrobotics/docs/tree/main/docs).
 For information about Hugo and how to develop locally, see the [README](https://github.com/viamrobotics/docs/blob/main/README.md).
 
+## Content Types
+
+The docs use the [Diátaxis Framework](https://diataxis.fr/) as the basis of the content structure with the following four content types:
+
+- **Explanation (conceptual)**: An understanding-oriented piece of content.
+  This content provides background knowledge on a topic and tends to be linked to from how-to guides and tutorials.
+  For example the [Robot Development Kit page](https://docs.viam.com/internals/rdk/) or the [Registry page](https://docs.viam.com/registry/).
+  It’s useful to have a real or imagined "Why?" question to serve as a prompt.
+
+  {{% expand "Click to view template" %}}
+
+  ```md
+  # Concept
+
+  Introductory paragraph.
+  Possibly containing further subsections or links to relevant conceptual information.
+
+  ## Subconcept
+
+  Paragraphs containing explanation. Add imagery as needed.
```

Comment:
```suggestion
  Paragraphs containing explanation. Add imagery as needed.
      - Provide context and address potential points of confusion.
      - Add real or hypothetical examples.
```
Maybe this elaboration would be helpful. Not sure about my "bullet points" here though..!